### DISCUSSION TITLE
Redirect to calling url after successful login

### DISCUSSION THREAD

**Discussion by shreeharsha-factly** [ory/kratos#971] [Q&A] [answered]
When I try to hit some protected routes _http://127.0.0.1:4455/kavach/users_ then it will be redirected to _http://127.0.0.1:4455/kavach/auth/login_.

Kratos config
````
selfservice:
  default_browser_return_url: http://127.0.0.1:4455/kavach/
    - http://127.0.0.1:4455
````

So after successful login how can I redirect to other protected url instead of `default_browser_return_url` set in kratos.

**Comment by vinckr** [comment, marked as answer]
Since you asked in [our chat](https://www.ory.sh/chat), I will post the answer here as well for the record.

>So once Iogin is done then  how to redirect back to that protected url instead of the default_browser_return_url set in kratos config?


>You can pass the return_to query parameter when initializing the flow
>That URL has to be whitelisted in the config

Thanks @zepatrik for answering this!

[Self-Service User Logout for Browser Applications](https://www.ory.sh/kratos/docs/self-service/flows/user-logout/#self-service-user-logout-for-browser-applications)
[Troubleshooting whitelisted URLs](https://www.ory.sh/hydra/docs/debug/#redirect-url-is-not-whitelisted)

**Reply by zepatrik** [reply]
As this question came up already a view times, would you mind improving the docs around that @vinckr ? Do you have suggestions or could you add a PR yourself @shreeharsha-factly ? Also, I will move this discussion to Kratos.
